runner.py

```diff
@@ -73,4 +73,5 @@ def configure_gradio_ui_envs() -> None:
             ssl_keyfile=config.ols_config.tls_config.tls_key_path,
             ssl_certfile=config.ols_config.tls_config.tls_certificate_path,
             ssl_keyfile_password=config.ols_config.tls_config.tls_key_password,
+            log_level="debug",
```

Comment:
what's the default level?  is this increasing the verbosity or decreasing it?

also seems like this should be configured via another field in https://github.com/openshift/lightspeed-service/blob/main/ols/app/models/config.py#L559-L560

Reply:
it is basically decreasing # messages in logs